db/migrate/20200614183600_add_achive_status_to_locations.rb

```diff
@@ -0,0 +1,5 @@
+class AddAchiveStatusToLocations < ActiveRecord::Migration[5.1]
+  def change
+    add_column :locations, :archived_at, :datetime
```

Comment:
do you want to have a `default: nil` here?

Reply:
Yeps, added!